pyproject.toml

```diff
@@ -12,6 +12,11 @@ readme = "README.md"
 authors = [{name = "Open Force Field Initiative", email = "info@openforcefield.org"}]
 license = {text = "MIT"}
 dynamic = ["version"]
+requires-python = ">= 3.11"
+dependencies = [
+  "pint >=0.24.0,<0.24.5",
```

Comment:
(blocking) Are there any tests that ensure this remains correct, or will we need to remember to manually change this when we update pint version ranges other places?

Reply:
Good catch, I'll add a workflow that installs from here